search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -287,6 +288,12 @@
     cqm/match-all
     (qm/->HasGranulesCondition (= "true" value))))
 
+(defmethod common-params/parameter->condition :has-granules-or-cwic
+  [_ _ _ value _]
+  (if (= "unset" value)
```

Comment:
"unset" was a holdover from the catalog-rest API for certain parameters. I don't think we need to support it for any new parameters.